core/src/epicli/data/common/ansible/playbooks/roles/backup/tasks/common/create_snapshot_archive.yml

```diff
@@ -0,0 +1,66 @@
+---
+# Invoke with (example):
+#- set_fact:
+#    snapshot_prefix: "haproxy_etc"
+#    snapshot_name: "20200526-102034"
+#    dirs_to_archive:
+#      - /etc/haproxy/
+#      - /etc/ssl/haproxy/
+#    files_to_archive:
+#      - /var/lib/rabbitmq/definitions/definitions-{{ snapshot_name }}.json
```

Comment:
Should such an example go to the documentation?

Reply:
I don't think so, this is not documentation for the end user, and this was never the intention. We don't currently have documentation for specific ansible task files.